nix/install

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # Vendored Nix installation script.
-# Retrieved 2020-01-08 from https://nixos.org/nix/install
+# Retrieved 2022-09-22 from https://nixos.org-2.10.3/nix/install
```

Comment:
Nit: This URL is invalid, I think the `-2.10.3` can be removed.

Reply:
`2.10.3` is not the latest version, so `https://nixos.org/nix/install` redirects to `2.11.1` (currently).